pkg/fspkg/bundle_package.go

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
Not a fan of `_` in Go file names. Maybe `parser.go` would be better?

Reply:
I don't like `parser.go` and I don't personally have a problem using `_` in Go file names... but I think `package.go` would be clearer and would remove the underscore, so I'll go for that.